### PR TITLE
Updated the Dockerfile to run in an alpine ruby container

### PR DESCRIPTION
**Pull request by jsa5593**
Changed the Dockerfile to run in alpine.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ruby-26
+FROM ruby:alpine as builder
 ENV HEIMDALLTOOLS_VERSION 1.3.6
```
Comment (rbclark) [inline Dockerfile:2]: This shouldn't be hardcoded to a version, I'd suggest we just drop this ENV completely.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 MAINTAINER rx294@nyu.edu
```
Comment (rbclark) [inline Dockerfile:3]: I'd probably go ahead and just remove the MAINTAINER tag on this since you just reworked the whole file.

**Comment by aaronlippold** [comment]
Did anyone test to make sure all the functions and modules work

**Comment by rbclark** [comment]
Yes, John ran through all of the options and verified that there are no obvious discrepancies between how it works inside and outside of Docker.

**Pull request merged**